code/scripting/api/objs/texture.cpp

```diff
@@ -117,6 +117,21 @@ ADE_FUNC(unload, l_Texture, NULL, "Unloads a texture from memory", NULL, NULL)
 	return ADE_RETURN_NIL;
 }
 
+ADE_FUNC(unloadRenderTarget, l_Texture, nullptr, "Unloads a texture's render target. Call this when done drawing to a texture, as it frees up resources.", nullptr, nullptr)
+{
+	texture_h* th;
+
+	if (!ade_get_args(L, "o", l_Texture.GetPtr(&th)))
+		return ADE_RETURN_NIL;
+
+	if (!th->isValid())
+		return ADE_RETURN_NIL;
+
+	bm_release_rendertarget(th->handle);
```

Comment:
Please verify before this that `bm_is_render_target` is true and cause a `LuaError` otherwise.